### DISCUSSION TITLE
Using G-Helper with XG Mobile

### DISCUSSION THREAD

**Discussion by GodIsAnAutomaton** [seerge/g-helper#3445] [General]
This came up a few times but I still seemed to have issues so wanted to put this here for anyone having trouble connecting the XG Mobile if you have a Flow series:

- G-Helper must be in Standard Mode first
- Then you can connect the XGM cable
- Restart G-Helper and the XGM will show as an option, after pressing it then you can lock the cable and any connected display or device should now be active
- Disconnecting XGM mode appears to be as simple as clicking the XGM box on G-Helper again to toggle off and it will ask to shutdown applications using it. That said even after doing that I'll get a pop-up that one of them crashes (i.e. Firefox whose process didn't fully stop) so to play it safe I shut down completely after toggling the XGM button off then I disconnect the cable and boot back up. This prevents getting errors about the XGM not being safely removed which I've received a few times after disconnecting after toggling XGM mode off.

Hope this helps. I know a little finicky but still beats the Armoury-Crate resource hog imo :D

**Comment by seerge** [comment]
@GodIsAnAutomaton hello, thanks for feedback!

Quick remarks:

1. You don’t need to restart G-Helper to make XGM button appear. Just hiding and showing main app window is enough, as it doesn’t monitor XGM in real-time to preserve resources, it does it only when window is shown.
2. Warning popup to close GPU apps before disconnecting XGM doesn’t stop any processes. It just warns you to do so :) Rest is up to you. However browsers usually survive GPU toggles without any problems (at least they don’t crash when Eco mode is selected)

**Reply by GodIsAnAutomaton** [reply]
Hi there, oh interesting. 1 did not appear to work for me and I had to completely exit out. Will attempt to reproduce issue and if anything reinstall. And good to know about 2, thanks for the heads up and all of your work on this great app!
